VimeoUpload.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,15 +2,15 @@
 <Workspace
```

Comment:
Sometimes I see `xcworkspacedata` and `xcsmblueprint` pop up. Do you know what those files are for?